enterprise-cfengine-guide/high-availability/installation-guide.markdown

```diff
@@ -503,7 +513,7 @@ HA fencing guide](https://access.redhat.com/documentation/en-us/red_hat_enterpri
 
    **IMPORTANT:** Copy over only the hashes, without the `SHA=` prefix.
 
-5. **On both nodes,** modify the */var/cfengine/masterfiles/controls/def.cf* and
+6. **On both nodes,** modify the */var/cfengine/masterfiles/controls/def.cf* and
```

Comment:
Instead of instructing people to modify two policy files, I think we should instead suggest defining the class via augments.

```
jq '.classes +=  { "enable_cfengine_enterprise_hub_ha": [ "any::" ] }' < masterfiles/def.json > masterfiles/def.json.new
mv masterfiles/def.json{.new,}
```